transport/http2_server.go

```diff
@@ -485,7 +486,11 @@ func (t *http2Server) WriteStatus(s *Stream, statusCode codes.Code, statusDesc s
 			Name:  "grpc-status",
 			Value: strconv.Itoa(int(statusCode)),
 		})
-	t.hEnc.WriteField(hpack.HeaderField{Name: "grpc-message", Value: statusDesc})
+	value := ""
+	if statusDesc != "" {
+		value = base64.StdEncoding.EncodeToString([]byte(statusDesc))
```

Comment:
base64 should things should have keys ending in `-bin` per http://www.grpc.io/docs/guides/wire.html